src/Header/HeaderRow.tsx

```diff
@@ -60,12 +64,15 @@ function HeaderRow<RecordType>({
           additionalProps = cell.column.onHeaderCell(column);
         }
 
+        const tdComponent = getComponent(['header', 'cell'], 'td');
+
         return (
           <Cell
             {...cell}
+            scope={column.title ? (cell.colSpan > 1 ? 'colgroup' : 'col') : null}
             ellipsis={column.ellipsis}
             align={column.align}
-            component={CellComponent}
+            component={column.title ? CellComponent : tdComponent}
```

Comment:
Cell in header is always `th` which should not replaced with `td`.

Reply:
But I only replace this if we don't have a title for that column. Because examples with empty cells in header use `td`, not `th`
![image](https://user-images.githubusercontent.com/69504277/193016900-fccddf52-d171-4dbb-8edb-9b67693b5fbe.png)

Reply:
OK. So let's use prop component also~~